src/helpers/getBasePath.ts

```diff
@@ -0,0 +1,2 @@
+export const getBasePath = (basePath?: string): string =>
+  basePath ? basePath.replace('/', '') + '/' : '';
```

Comment:
I think I have it in two places, I wanted to avoid duplication.

Reply:
I've actually moved this to useAlternateLinks file since I only use it there.

Reply:
Af first I though about using it in `useAlternateLins` file, however, because I'm also using this in `useCanoncalUrl` file I believe it's best to have it on it's own file.